### PR TITLE
implementation of timestamp support + tests

### PR DESCRIPTION
**Pull request by micnovak**

### CODE REVIEW

File: src/confd_gnmi_common.py [hunk 1]
```diff
@@ -277,3 +277,21 @@ def stream_mode_str_to_int(mode: str, no_error=False) -> int:
...
+    """
+    Get the current timestamp in nanoseconds.
+    Returns:
+        int: The current timestamp in nanoseconds.
+    """
+    return int(time.time_ns())
+
+def get_time_string(time_ns) -> str:
+    """
+    Get the  timestamp in nanoseconds.
```
Comment (jomiklos) [inline src/confd_gnmi_common.py:292]: its not returning string in nanoseconds at all?

File: tests/client_server_test_base.py [hunk 2]
```diff
@@ -201,14 +205,17 @@ def read_subscribe_responses(responses, read_count=-1):
...
+                time_after = get_timestamp_ns()
+                log.debug("response=%s response_count=%i time_after=%i", response,
+                          response_count, time_after)
+                response_time_ms = time_after/1000000
                 if response.sync_response:
                     log.debug("sync_response")
                     assert response_count == 1  # sync expected only after first response
                 else:
                     response_count += 1
+                    assert (time_before <= response.update.timestamp and
```
Comment (jomiklos) [inline tests/client_server_test_base.py:217]: can this handle/survive client/server in different timezones or incorrectly set time?
why do we need "validation" here?

**Review by jomiklos — APPROVED** [review]

File: src/confd_gnmi_common.py (hunk 1 above)
Comment (micnovak) [reply src/confd_gnmi_common.py]: Comment updated ....

File: tests/client_server_test_base.py (hunk 2 above)
Comment (micnovak) [reply tests/client_server_test_base.py:217]: yes, time is in UT ns, so it is local independent.

**Review by martin-volf — APPROVED** [review]

**Pull request merged**